src/workerd/server/pyodide.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+#pragma once
+
+#include <workerd/api/pyodide/pyodide.h>
+#include <workerd/io/compatibility-date.capnp.h>
+#include <workerd/jsg/jsg.h>
+
+#include <kj/array.h>
+#include <kj/compat/http.h>
+#include <kj/filesystem.h>
+#include <kj/string.h>
+#include <kj/timer.h>
+
+namespace workerd::server {
+
+// Helper functions for bundle file operations
+kj::Path getPyodideBundleFileName(kj::StringPtr version);
```

Comment:
Nit: This and a few more functions are only used in pyodide.c++, no need to add them to the header